lib/mayaUsd/render/vp2RenderDelegate/basisCurves.cpp

```diff
@@ -1264,9 +1264,9 @@ void HdVP2BasisCurves::_UpdateDrawItem(
                 int  primitiveStride = 0;
 
                 const MColor& color
-                    = (_selectionStatus != kUnselected
-                           ? drawScene.GetSelectionHighlightColor(_selectionStatus == kFullyLead)
-                           : drawScene.GetWireframeColor());
+                    = (_selectionStatus != kUnselected ? drawScene.GetSelectionHighlightColor(
+                           _selectionStatus == kFullyLead ? nullptr : "curve")
+                                                       : drawScene.GetWireframeColor());
```

Comment:
Should this little bit of logic deciding which color to use based on the selection status move into a common location? I'm not sure if it makes sense on drawScene, maybe if we do a shared parentClass for rprims this logic could live there.

Reply:
Yes, I think I will address that as part of my future work on points.